internal/objectstatus/ingress.go

```diff
@@ -74,6 +75,15 @@ func (is *ingressStatus) run(ctx context.Context) (ObjectStatus, error) {
 	}
 
 	for _, backend := range backends {
+		if backend.ServicePort.String() == "use-annotation" {
+			albAction := ingressAlbActionAnnotation + backend.ServiceName
+			if _, ok := ingress.Annotations[albAction]; !ok {
+				status.SetError()
+				status.AddDetailf("Backend refers to annotations %q which does't exist", albAction)
```

Comment:
I think we should grab the information out of the annotations and write it to the config card.